cli/test/org/partiql/cli/CliTest.kt

```diff
@@ -155,6 +155,30 @@ class CliTest {
         assertEquals("{a:1}\n{b:1}\n", actual)
     }
 
+    @Test
+    fun withSelectIn() {
+        makeCli(
+            query = "SELECT age FROM << { 'name': 'John', 'age': 22 } >> WHERE name IN (SELECT name FROM <<{ 'name': 'John' }>>)",
+            output = FileOutputStream(testFile),
+            outputFormat = OutputFormat.ION_TEXT
+        ).run()
+        val actual = testFile.bufferedReader().use { it.readText() }
+
+        assertEquals("{age:22}\n", actual)
+    }
+
+    @Test
+    fun withSelectInButEmpty() {
+        makeCli(
+            query = "SELECT age FROM << { 'name': 'John', 'age': 22 } >> WHERE name IN (SELECT name FROM <<{ 'name': 'john' }>>)",
+            output = FileOutputStream(testFile),
+            outputFormat = OutputFormat.ION_TEXT
+        ).run()
+        val actual = testFile.bufferedReader().use { it.readText() }
+
+        assertEquals("\n", actual)
+    }
+
```

Comment:
Please add these tests to `EvaluatorTestSuite`--instead they are not needed here if the exist there.  

Also, please add a simpler versions of the test case which will be easier to understand the actual feature being tested:

These all must return `true`:

```
`foo` in << { 'foo': } >>
`foo` in [ { 'foo': } ]
`foo` in SEXP({ 'foo': })
```
And these must all return false:

```
`bar` in << { 'foo': } >>
`bar` in [ { 'foo': } ]
`bar` in SEXP({ 'foo': })
```